src/lib/transport.js

```diff
@@ -137,7 +142,12 @@ function * listen (plugin, {
         destinationAccount,
         destinationAmount,
         fulfill: function () {
-          return plugin.fulfillCondition(transfer.id, fulfillment)
+          return retryPromise({
+            callback: () => plugin.fulfillCondition(transfer.id, fulfillment),
```

Comment:
I still think we _might_ want to catch the final errors defined in the Ledger Plugin Interface so that we don't retry those